### PR TITLE
Prevent default on anchor links

### PR DESCRIPTION
**Pull request by jonalu**
I think for most cases it's preferable to disable default behavior on anchor links events, to avoid scroll-to-top issue.

### CODE REVIEW

File: index.jsx [hunk 1]
```diff
@@ -29,18 +29,23 @@
                 items: []
             };
         },
-        goTo: function(page) {
+        goTo: function(page,e) {
```
Comment (eliseumds) [inline index.jsx]: Could you please match styles? One space after comma, and spaces before and after parentheses.

File: index.jsx [hunk 2]
```diff
@@ -29,18 +29,23 @@
...
+            if(e){
```
Comment (eliseumds) [inline index.jsx]: Break line after this IF block.

File: index.jsx [hunk 3]
```diff
@@ -29,18 +29,23 @@
...
+            e.preventDefault();
             var page = this.state.currentPage;
 
             if (page < this.props.max) {
                 this.goTo(page + 1);
             }
         },
-        onClickPrev: function() {
+        onClickPrev: function(e) {
+            e.preventDefault();
```
Comment (eliseumds) [inline index.jsx:50]: Break one line after.

**Comment by jonalu** [comment]
Fixed the issues you mentioned. Sorry about the mismatch in style, I did all that in a hurry.

**Comment by eliseumds** [comment]
@jonalu, awesome, thanks! :+1:

**Pull request merged**